src/common/schema.graphql.ts

```diff
@@ -87,6 +87,11 @@ export default gql`
 		DESC
 	}
 
+	enum EmailType {
+		ACCEPTANCE
```

Comment:
will there be other email types, eg. Confirm, Register, etc. in the future?

Reply:
It might be worth leaving this as a variable string instead of an enum if so